teuthology/run.py

```diff
@@ -224,7 +224,7 @@ def get_initial_tasks(lock, config, machine_type):
             {'internal.archive': None},
             {'internal.coredump': None},
             {'internal.sudo': None},
-            {'internal.syslog': None},
+            {'internal.syslog': overrides.get('ceph', {})},
```

Comment:
```suggestion
            {'internal.syslog': config.get('syslog', {})},
```

I mentioned using overrides but I now see we don't particularly need to use that. See further below: